cornice_swagger/swagger.py

```diff
@@ -383,6 +393,86 @@ def _ref(self, param, depth, base_name=None):
         return {'$ref': pointer}
 
 
+class ResponseHandler(object):
+    """Handles swagger response definitions."""
+
+    json_pointer = '#/responses/'
+
+    def __init__(self, definition_handler=DefinitionHandler(), ref=False):
+        """
+        :param definition_handler:
+            Callable that handles swagger definition schemas.
+        :param ref:
+            Specifies the ref value when calling from_xxx methods.
+        """
+
+        self.responses = {}
```

Comment:
I agree, I'm not sure if we should use definitions because there is a section named definitions on the swagger document. Maybe call it `response_registry`?